recipes/r-rvcg/meta.yaml

```
@@ -0,0 +1,130 @@
+{% set version = '0.22.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rvcg
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/Rvcg_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/Rvcg/Rvcg_{{ version }}.tar.gz
+  sha256: dbaa2e76fd5f7e3d09f9cc326eea2b225be1be49c4f21b403d0bd3c3f114399b
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+  # no skip
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  
+  missing_dso_whitelist:
+    - '*/R.dll'
+
+# Suggests: Morpho, rgl
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+
+  host:
+    - r-base
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-rcppeigen
+    - _openmp_mutex                 # [linux]
```

Why this dependency? More typical is:

```suggestion
    - libgomp                       # [linux]
```